config/settings.yml

```diff
@@ -1109,6 +1109,11 @@
       :thread_shutdown_timeout: 10.seconds
     :event_catcher:
       :defaults:
+        :flooding_monitor_enabled: false
+        :flooding_monitor_window: 5.minutes
+        :flooding_monitor_slot_size: 5.seconds
+        :flooding_monitor_threshold: 20
+        :flooded_log_count: 20
```

Comment:
Do we need all of these options?  Can we get away with sensible defaults and not concern the user about the details?